<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meristem Phenotyper 3D - Batch Processor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_parameters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QGroupBox" name="groupBox_contour_fit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Contour Fit</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cf_type_input">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Threshold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Active Contours</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Weight Factor</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="cf_weight_factor_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.800000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Smoothing</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="cf_smooth_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_prim_ident">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>251</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Primordia Identification</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh Smoothing</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="mesh_smooth_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh Relaxation</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="relaxation_factor_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <double>2.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Curvature Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="curvature_type_threshold">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Mean</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaussian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Max</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Curvature Threshold</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="curvature_threshold_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimal Size [%]</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="min_feature_size_percent_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_paraboloid_fit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Fit Meristem with Paraboloid</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>20</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_total">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>470</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_save_location">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt; enter directory &lt;&lt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_save_location">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>23</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_files">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>441</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QListWidget" name="listWidget_filenames">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>311</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add files with drag &amp; drop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_file">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/remove-icon-png-24.png</normaloff>../../Downloads/remove-icon-png-24.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
